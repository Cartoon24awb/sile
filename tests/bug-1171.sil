\begin[class=jplain,layout=tate]{document}
\set[parameter=font.family,value=Noto Serif CJK JP,makedefault=true]
\define[command=emoji]{\font[family=Twemoji Mozilla]{\process}}
いいですよ。\emoji{😉} \emoji{🤣}
\end{document}
