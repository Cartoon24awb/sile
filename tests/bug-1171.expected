Set paper size 	595.275597	841.8897729
Begin page
Mx 	542.4079
My 	107.6592
Set font 	Noto Serif CJK JP;10;400;;normal;;TTB
T	64423 w=10.0000	(„ÅÑ)
My 	117.6592
T	64423 w=10.0000	(„ÅÑ)
My 	127.6592
T	64458 w=10.0000	(„Åß)
My 	137.6592
T	64444 w=10.0000	(„Åô)
My 	147.6592
T	64491 w=10.0000	(„Çà)
My 	157.6592
T	59878 w=10.0000	(„ÄÇ)
Push color	1.0000	0.8000	0.3020
My 	167.6592
Set font 	Twemoji Mozilla;10;400;;normal;;TTB
T	7876 a=10.0000	()
Pop color
Push color	0.4000	0.2706	0.0000
Mx 	10.0000
T	7893 a=10.0000 7894 a=10.0000	(üòâ)
Pop color
Push color	1.0000	0.7961	0.2980
Mx 	542.4079
My 	180.8077
T	9105 a=10.0000	()
Pop color
Push color	0.3961	0.2784	0.1059
T	9106 a=10.0000	()
Pop color
Push color	1.0000	1.0000	1.0000
T	9107 a=10.0000	()
Pop color
Push color	0.3961	0.2784	0.1059
T	9108 a=10.0000	()
Pop color
Push color	0.3922	0.6667	0.8667
T	9109 w=10.0000	(ü§£)
Pop color
Mx 	298.2129
My 	781.9286
Set font 	Noto Serif CJK JP;10;400;;normal;;LTR
T	18 w=5.3900	(1)
End page
Finish
